app/views/user_mailer/shared/_non_discrimination_language.html.erb

```diff
@@ -0,0 +1,113 @@
+<html lang="es">
+¡Importante! Esto tiene información importante sobre su seguro de salud. Si usted quiere la información traducida a su propio idioma, llame al 1-888-813-9220.
```

Comment:
can 1-888-813-9220 be a settings entry and helper method? 

non_discrimination_phone_number